framework/TSA/ARMA.py

```diff
@@ -25,11 +25,11 @@
 statsmodels = importerUtils.importModuleLazy('statsmodels', globals())
 
 import Distributions
-from .TimeSeriesAnalyzer import TimeSeriesAnalyzer
+from .TimeSeriesAnalyzer import TimeSeriesAnalyzer, TimeSeriesGenerator, TimeSeriesCharacterizer
```

Comment:
We can omit the base import here, right? Similarly for the other TSA files.

Reply:
I believe we need it for the `handleInput()` method we use the base class to call that method.

Reply:
Interesting. Would this work as a super call instead of directly calling TimeSeriesAnalyzer.handleInput, or does that cause inheritance resolution issues?